scim/users.go

```diff
@@ -88,6 +89,18 @@ func (a UsersAPI) Delete(userID string) error {
 	return a.client.Scim(a.context, http.MethodDelete, userPath, nil, nil)
 }
 
+func (a UsersAPI) DeleteRepos(userID string) error {
```

Comment:
This does not belong here, as we're migrating the provider to Go SDK https://github.com/databricks/databricks-sdk-go/blob/main/service/scim/api.go#L827-L985

move this code to delete callback body